packages/core/src/App/Containers/Layout/header/dashboard-platform-header.jsx

```diff
@@ -8,7 +8,7 @@ import { BinaryLink } from 'App/Components/Routes';
 import PropTypes from 'prop-types';
 import ToggleMenuDrawer from 'App/Components/Layout/Header/toggle-menu-drawer.jsx';
 import { connect } from 'Stores/connect';
-import platform_config from 'App/Constants/platform-config';
+import platform_config from 'App/Constants/platform-config.ts';
```

Comment:
I'm not sure about this, does this throw an error if you don't define the file type?

Reply:
importing ts files to js files leads to this eslint issue. ts to ts works fine without file type.
<img width="974" alt="Screenshot 2023-08-30 at 1 09 05 PM" src="https://github.com/binary-com/deriv-app/assets/122449658/5ac121bb-c5e9-4834-8394-e8c09068c6c2">